src/app-layer-htp-file.h

```diff
@@ -25,7 +25,15 @@
 #ifndef __APP_LAYER_HTP_FILE_H__
 #define __APP_LAYER_HTP_FILE_H__
 
+typedef struct HtpContentRange_ {
+    int64_t start;
+    int64_t end;
+    int64_t size;
+} HtpContentRange_t;
```

Comment:
we don't use the _t suffix for types in Suri

Reply:
ok